code/datums/diseases/advance/symptoms/shedding.dm

```diff
@@ -28,12 +28,14 @@ BONUS
 	..()
 	if(prob(SYMPTOM_ACTIVATION_PROB))
 		var/mob/living/M = A.affected_mob
+		var/mob/living/carbon/human/H = M
+		if(istype(H) && !H.get_organ("head"))
+			return
 		to_chat(M, "<span class='warning'>[pick("Your scalp itches.", "Your skin feels flakey.")]</span>")
 		if(ishuman(M))
-			var/mob/living/carbon/human/H = M
+			var/obj/item/organ/external/head/head_organ = H.get_organ("head")
```

Comment:
This could probably be simplified, but I'm struggling to see a way that makes sense while keeping the messages sane. Basically I was thinking the `istype(H)` and `ishuman(M)` could be merged somehow since they are doing the same thing.

Reply:
I ended up ditching the living check on this and only making it apply to humans.